src/main.js

```diff
@@ -173,6 +175,7 @@ class MiniMediaPlayer extends LitElement {
         artwork=${config.artwork} ?has-artwork=${artwork} state=${this.player.state}
         ?flow=${config.flow} ?collapse=${config.collapse}
         content=${this.player.content}
+        ?rtl=${this._computeRTL(this.hass)}
```

Comment:
What do you think about making this check once inside `set hass()` instead and save the result as a boolean in a member variable (this.rtl) instead? Then we could resure that variable throughout the rendering logic. Like below.

https://github.com/kalkih/mini-media-player/blob/bc848c6e59cc100e8f8b661b036ae552df148b8a/src/main.js#L57-L61